pkg/app/mysql.go

```diff
@@ -0,0 +1,227 @@
+package app
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	"github.com/kanisterio/kanister/pkg/format"
+	"github.com/kanisterio/kanister/pkg/kube"
+	"github.com/kanisterio/kanister/pkg/log"
+	"github.com/kanisterio/kanister/pkg/testutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+type chartInfo struct {
+	helmrepoURL   string
+	helmChartName string
+	helmAppname   string
+}
+
+type MysqlDB struct {
+	cli       kubernetes.Interface
+	namespace string
+	password  string
+	chart     chartInfo
+}
+
+func NewMysqlDB(helmrepoURL, helmChart, helmAppname string) App {
+	return &MysqlDB{
+		password: "mysecretpassword",
+		chart: chartInfo{
+			helmrepoURL:   helmrepoURL,
+			helmChartName: helmChart,
+			helmAppname:   helmAppname,
+		},
+	}
+}
+
+func (mdb *MysqlDB) Init(ctx context.Context) error {
+
+	cfg, err := kube.LoadConfig()
+	if err != nil {
+		return nil
+	}
+
+	mdb.cli, err = kubernetes.NewForConfig(cfg)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (mdb *MysqlDB) Install(ctx context.Context, namespace string) error {
+
+	mdb.namespace = namespace
+
+	cli := testutil.NewHelmClient()
+	log.Print("Adding repo.")
```

Comment:
@viveksinghggits let's use the following format for logging accross all the apps 
`log.Info/Debug/Error().Print("msg", field.M{"app": app.name, additional_fields... })`
Let me know what you think.